presto-jmx/src/main/java/com/facebook/presto/connector/jmx/JmxMetadata.java

```diff
@@ -90,14 +128,17 @@ public JmxTableHandle getTableHandle(ConnectorSession session, SchemaTableName t
             MBeanInfo mbeanInfo = mbeanServer.getMBeanInfo(objectName.get());
 
             ImmutableList.Builder<JmxColumnHandle> columns = ImmutableList.builder();
-            columns.add(new JmxColumnHandle(connectorId, "node", VARCHAR));
-            for (MBeanAttributeInfo attribute : mbeanInfo.getAttributes()) {
-                if (!attribute.isReadable()) {
-                    continue;
-                }
-                columns.add(new JmxColumnHandle(connectorId, attribute.getName(), getColumnType(attribute)));
-            }
-            return new JmxTableHandle(connectorId, objectName.get().toString(), columns.build());
+            columns.add(new JmxColumnHandle(connectorId, NODE_COLUMN_NAME, VARCHAR));
+
+            // Since this method is being called on all nodes in the cluster, we must ensure (by sorting)
```

Comment:
Previously this method was being called only on master node in `ConnectorMetadata::getTableHandle`. Now it is also called in dumping thread. Since we store dump rows in ArrayList of `columnValues` (not in Map `columnName -> columnValue`), we need it to be in the same order on all nodes.